src/lib.rs

```diff
@@ -286,6 +286,11 @@ impl<L: AnalysisLoader> AnalysisHost<L> {
         })
     }
 
+    /// Returns all the imports in the analysis data.
+    pub fn find_all_imports(&self) -> AResult<Vec<Import>> {
```

Comment:
I would call this `all_imports` - it's not really finding anything, just dumping everything at once.